### ISSUE TITLE
[Accessibility][CSS][PDF Viewer] Dialog links within paragraph text have no visual distinction from surrounding text other than their color 

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1737]
**Steps to Reproduce**
1. Open the page in Google Chrome.
2. Activate the menu (3 dots).
3. Activate the "Legal Notices" dialog.
4. View links in text.

**Current Behavior**
Links are blue (#4C8DDA), which does not meet a 3:1 contrast ratio with surrounding gray text, and no other visual distinctions are made from surrounding text. Users with low vision may be unable to perceive these links as distinct from the rest of the text.

**Expected behavior/code**
Links within copy are best communicated to users by using an underline. Using only a difference in color is acceptable as long as it meets a 3:1 color contrast ratio with surrounding text.

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Users with low vision and/or limited perception of color

**WCAG SC**
1.4.1 Use of Color

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES

**Comment by glasert** [comment]
This issue won't be fixed as the acrobat team is working on a modern viewer which addresses accessibility and will replace the current viewer in 2022. Therefore we defer and close this issue
